src/System.Runtime.Extensions/src/System/IO/StringWriter.cs

```diff
@@ -159,11 +159,29 @@ public override void Write(string value)
             }
         }
 
+        public override void Write(StringBuilder value)
+        {
+            if (GetType() != typeof(StringWriter))
+            {
+                // This overload was added after the Write(StringBuilder, ...) overload, and so in case
```

Comment:
"after the Write(StringBuilder, ...) overload" =>
"after the Write(char[], ...) overload"